packages/manager/modules/octavia-load-balancer/src/onboarding/no-private-network/constants.js

```diff
@@ -0,0 +1,48 @@
+export const PRIVATE_NETWORK_HELP = {
+  DEFAULT:
+    'https://docs.ovh.com/fr/publiccloud/network-services/networking-concepts/',
```

Comment:
default link can be for English.
```suggestion
    'https://docs.ovh.com/gb/en/publiccloud/network-services/networking-concepts/',
```

Reply:
Done!